src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,265 @@
+package com.manardenza.console;
+
+import com.manardenza.controller.HotelController;
+import com.manardenza.controller.ReservationController;
+import com.manardenza.controller.UserController;
+import com.manardenza.entity.Hotel;
+import com.manardenza.entity.Reservation;
+import com.manardenza.entity.Room;
+
+import java.io.IOException;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+import static com.manardenza.console.VisualUserMenu.inputDataFromUser;
+import static com.manardenza.console.VisualUserMenu.outputDataInConsole;
+import static com.manardenza.console.VisualUserMenu.outputListInConsole;
+import static com.manardenza.console.VisualUserMenu.outputSplitLine;
+import static com.manardenza.console.VisualUserMenu.readerMenuFromConsole;
+import static com.manardenza.console.VisualUserMenu.showUserMenu;
+
+public class ContentsUserMenu {
+
+    private HotelController hotelController;
+    private ReservationController reservationController;
+    private UserController userController;
+
+    public ContentsUserMenu(HotelController hotelController,
+                            ReservationController reservationController, UserController userController) {
+        this.hotelController = hotelController;
+        this.reservationController = reservationController;
+        this.userController = userController;
+    }
+
+    public void mainMenu() throws IOException {
+        outputSplitLine();
+        outputDataInConsole("\t\tWelcome to manardenza.com");
+        outputSplitLine();
+        outputDataInConsole("\n\tAuthorization");
+        showUserMenu("Main menu");
+        int actionMain = readerMenuFromConsole("Choose action: ");
+        switch (actionMain) {
+            case (1):
+                registrationUserMenu();
+                break;
+            case (2):
+                outputDataInConsole("\tBuy!");
+                System.exit(0);
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Remove this call to "exit" or ensure it is really required. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1147)